games/lockpicking/src/styles/pages/score-tabs.css

```diff
@@ -0,0 +1,133 @@
+.block {
```

Comment:
Remove unused styles.
Also, styles like ".block" so bad. Because it is so abstract